readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -177,6 +163,9 @@ def append_conf(self, **__):
         except IOError:
             raise ProjectConfigurationError(ProjectConfigurationError.NOT_FOUND)
 
+        # Create an index file if there is None.
+        self.create_index(extension='rst')
```

Comment:
Considering that we are removing all these auto-generated config files, is this something that we still want?

If we are now requiring a `conf.py` in the project, it means that the user have already ran `sphinx-quickstart` which already generates a `index.rst`.

Reply:
This is another problem. Previously sphinx-quickstart used to generated another filename for index.

The issue to track this is https://github.com/rtfd/readthedocs.org/issues/1800